### ISSUE TITLE
pylint_runner fails in docker

### ISSUE THREAD

**Issue by FormMe** [MasterOdin/pylint_runner#8]
I try to run pylint_runner in docker, but then i get this error. If I run it without docker everything is ok.

`> docker-compose run my-image pylint_runner`

```
Starting my-image ... done

Traceback (most recent call last):
  File "/usr/local/bin/pylint_runner", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/pylint_runner/main.py", line 179, in main
    runner.run(output, error)
  File "/usr/local/lib/python3.6/site-packages/pylint_runner/main.py", line 169, in run
    run = pylint.lint.Run(self.args + pylint_files, exit=False)
TypeError: __init__() got an unexpected keyword argument 'exit'
```

**Comment by MasterOdin** [comment]
Yeah, it's because in the latest version of pylint, they changed the name of a parameter I was using which was fixed by the now merged #6. I'm going to go through and clean-up some stuff, cut a release `0.5.3` which will be the last release to support Python2 and then cut a `0.6.0` that is Python3 only.

**Comment by MasterOdin** [comment]
Wow did I lose track of this project. 0.6.0 released which should fix this issue. Ironically, the latest versions of pylint are now usable on pylint_runner 0.5.x as well since they've gone back to the `exit` arg.
